### ISSUE TITLE
Allow for horizontal splits

### ISSUE THREAD

**Issue by jeetsukumaran** [dahu/vim-lotr#2]
Great plugin!

Is it possible to allow for horizontal splits?

Basically, all this would need is setting:

```
 let openpos = 'botright '
```

in the function `s:OpenWindow()` based on some global option variable value.

Currently the code is:

```
  let openpos = g:lotr_left ? 'topleft vertical ' : 'botright vertical '
```

Maybe instead of `g:lotr_left`, something like `g:lotr_position` which can take on values of "top", "bottom", "left", "right"?

I would patch and submit a pull request, but (a) current HEAD requires a more recent Vim than I have available (I use MacVim) ...

**Comment by dahu** [comment]
Fixed in b68c8dd

**Comment by jeetsukumaran** [comment]
Thanks!
